<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>1024</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Калькулятор оплаты за горячую воду</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="meterDisplayGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Раздел показаний счетчика</string>
    </property>
    <property name="title">
     <string>Показания счетчика</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>482</width>
       <height>175</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="oldFrom40To44Label">
        <property name="text">
         <string>40 - 44 C:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="oldFrom45To49Label">
        <property name="text">
         <string>45 - 49 C:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="oldGreaterThen50Label">
        <property name="text">
         <string>&gt; 50 C:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" alignment="Qt::AlignHCenter">
       <widget class="QDoubleSpinBox" name="oldGreaterThen50SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;&gt; 50 C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="oldLessThen40SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;&lt; 40 C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="oldLessThen40Label">
        <property name="text">
         <string>&lt; 40 C:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="oldFrom45To49SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;45 - 49 C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="oldFrom40To44SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;40 - 44C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="currentFrom40To44SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;40 - 44C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="currentFrom45To49SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;45 - 49 C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="currentGreaterThen50SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;&gt; 50 C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="oldMeterDataLabel">
        <property name="text">
         <string>Было</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="newMeterDataLabel">
        <property name="text">
         <string>Стало</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="meterCellLabel">
        <property name="text">
         <string>Ячейка</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="currentLessThen40SpinBox">
        <property name="toolTip">
         <string>Сюда вносить данные из ячейки &quot;&lt; 40 C&quot;</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="differenceLabel">
        <property name="text">
         <string>Разница</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="from40To44VolumeSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Сколько потреблено воды &quot;40 - 44C&quot;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="from45To49VolumeSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Сколько потреблено воды &quot;45 - 49 C&quot;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3" alignment="Qt::AlignHCenter">
       <widget class="QDoubleSpinBox" name="greaterThen50VolumeSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Сколько потреблено воды &quot;&gt; 50 C&quot;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="lessThen40VolumeSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Сколько потреблено воды &quot;&lt; 40C&quot;</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tariffsGroupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Раздел тарифов</string>
    </property>
    <property name="title">
     <string>Тарифы</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>315</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="heatedWaterTariffLabel">
        <property name="text">
         <string>На горячую воду:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="heatedWaterTariffSpinBox">
        <property name="toolTip">
         <string>Сюда вносить значение тарифа на горячую воду</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="coldWaterTariffLabel">
        <property name="text">
         <string>На холодную воду:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="coldWaterTariffSpinBox">
        <property name="toolTip">
         <string>Сюда вносить значение тарифа на холодную воду</string>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="toPayGroupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Раздел расчитанной оплаты</string>
    </property>
    <property name="title">
     <string>К оплате</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>313</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="toPayLabel">
        <property name="text">
         <string>Всего:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="toPaySpinBox">
        <property name="toolTip">
         <string>Сколько нужно заплатить за всё</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="giocLabel">
        <property name="text">
         <string>Вносить на ГІОЦ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="giocSpinBox">
        <property name="toolTip">
         <string>Значение отсюда вводить на ГІОЦ</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Панель меню</string>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="newSession"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить текущую сессию</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="toolTip">
    <string>Выйти из программы</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть существующию сессию внесения показаний</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="newSession">
   <property name="text">
    <string>Новый</string>
   </property>
   <property name="toolTip">
    <string>Начать новую сессию внесения показаний</string>
   </property>
  </action>
  <action name="showStatistics">
   <property name="text">
    <string>Показать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
